.travis.yml

```diff
@@ -1,87 +1,81 @@
 language: php
 
-sudo: false
+sudo: true
+
+addons:
+  firefox: "47.0.1"
+  postgresql: "9.4"
+  apt:
+    packages:
+      - openjdk-8-jre-headless
+
+services:
+  - mysql
 
 cache:
   directories:
     - $HOME/.composer/cache
+    - $HOME/.npm
 
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env: MOODLE_BRANCH=MOODLE_31_STABLE DB=pgsql
-    - php: 5.6
-      env: MOODLE_BRANCH=MOODLE_31_STABLE DB=mysqli
```

Comment:
Not testing with Moodle 3.1 as the grunt tasks in moodle-plugin-ci are not supported in 3.1.